src/scripts.js

```diff
@@ -3,68 +3,80 @@ const users = document.querySelector('.users')
-    ${user.getFirstName()}'s goal is ${averagePercent}% of the average
-    </p>
-    `)
+
+function createFriendsList(user) {
+  const friendsNames = document.querySelector('.friends-names')
+  const friendsList = user.userFriends.map(friends => userRepo.getUserByID(friends))
+  friendsList.forEach(friend => friendsNames.insertAdjacentHTML('beforeEnd', `<p>${friend.name.split(' ')[0]}</p>`))
+}
+
+function createStepGoal(user) {
+  const stepGoal = document.querySelector('.step-goal')
+  stepGoal.innerHTML =
+  `<p>${user.getFirstName()}'s goal is  ${user.dailyStepGoal} steps per day, and the average is  ${userRepo.getAverageStepGoal()} steps per day.
+  <p class="ahead-or-behind"></p>`
+}
+
+function compareStepGoal(user) {
+  const difference = userRepo.getAverageStepGoal() - user.dailyStepGoal
+  const absoluteValue = Math.abs(difference)
+  const aheadOrBehind = document.querySelector('.ahead-or-behind')
+  difference > 0 ?  aheadOrBehind.innerText = `Your goal is ${absoluteValue} steps greater than the average!` : aheadOrBehind.innerText = `Your goal is ${absoluteValue} steps less than the average!`
+}
+
+function displayUserInfo(user) {
+  showInfoCard(user)
+  createFriendsList(user)
+  createStepGoal(user)
+  compareStepGoal(user)
 }
 
 function makeHydration(user) {
-  let newHydration = new Hydration(hydrationData, user)
+  const newHydration = new Hydration(hydrationData, user)
   showHydrationCard(newHydration)
 }
 
 function showHydrationCard(newHydration) {
   let weeksHydroData = newHydration.getWeekOfHydroData(todaysDate)
-  console.log(weeksHydroData)
   hydration.innerHTML = `
-    <section class="hydration-data"><p>Hydration Average: ${newHydration.getAverageDailyOunces()} oz</p></section>
-    <section class="weekly-hydration"><p>Todays Hydration: ${newHydration.getOuncesForSpecificDay(todaysDate)} oz</p></section>
-    <section class="hydration-data">
-    <p class="box-text">Yesterday's Hydration: ${weeksHydroData[0].numOunces} oz</p>
-    <p class="box-text">2 days ago: ${weeksHydroData[1]}oz</p>
-    <p class="box-text">3 days ago: ${weeksHydroData[2]}oz</p>
-    <p class="box-text">4 days ago: ${weeksHydroData[3]}oz</p>
-    <p class="box-text">5 days ago: ${weeksHydroData[4]}oz</p>
-    <p class="box-text">6 days ago: ${weeksHydroData[5]}oz</p>
-    <p class="box-text">7 days ago: ${weeksHydroData[6]}oz</p>
-    </section>
-                         `
+       <section class="hydration-data"><p>Hydration Average: ${newHydration.getAverageDailyOunces()} oz</p></section>
+       <section class="weekly-hydration"><p>Todays Hydration: ${newHydration.getOuncesForSpecificDay(todaysDate)} oz</p></section>
+       <section class="hydration-data">
+       <p class="box-text">Yesterday's Hydration: ${weeksHydroData[0].numOunces} oz</p>
+       <p class="box-text">2 days ago: ${weeksHydroData[1]}oz</p>
+       <p class="box-text">3 days ago: ${weeksHydroData[2]}oz</p>
```

Comment:
I think we'll need to do ${weeksHydroData[i].numOuces} for each of these to get the right value.
We can also do ${weeksHydroData[i].date} instead of "days ago" for a little sanity check.